ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,52 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    dtypes, arrays, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            num_arrays=helpers.ints(min_value=1, max_value=6),
+            ret_shape=True,
+            min_num_dims=2,
+            min_dim_size=2,
+            shared_dtype=True,
+        )
+    )
+    axis = draw(helpers.get_axis(shape=shape))
+    return dtypes, arrays, axis
+
+
+# concat_from_sequence
+@handle_test(
+    fn_tree="functional.ivy.experimental.concat_from_sequence",
+    dtypes_arrays_axis=_concat_from_sequence_helper(),
```

Comment:
Alternatively, instead of using a composite strategy to generate the arrays and axis separately, you could call `dtype_values_axis` straight away here, keeping the same arguments you are passing to `dtype_and_values` and adding also the `valid_axis` and `force_int_axis` set to True to make sure the axis is in right format.